app/static/js/userProfile.js

```diff
@@ -249,41 +272,48 @@ $(document).ready(function(){
       let bgDate =  bgDateInput.val()
       let bgStatus = $("[data-id=" + bgCheckType + "]").val()
 
-      if (bgStatus == '') {
-        bgStatusInput.focus()
-        bgStatusInput.addClass("invalid");
-        window.setTimeout(() => bgStatusInput.removeClass("invalid"), 1000);
-        $(this).prop("disabled", false);
-        return false
-      }
+        if (bgStatus == '') {
+          bgStatusInput.focus()
+           $('.form-select').each(function() {
+                bgStatusInput[0].setCustomValidity("Please enter a status");
+                bgStatusInput[0].reportValidity();
+          });
+          $(this).prop("disabled", false);
+          return false
+        }
 
-      if (bgDate == ''){
-        bgDateInput.focus()
-        bgDateInput.addClass("invalid");
-        window.setTimeout(() => bgDateInput.removeClass("invalid"), 1000);
-        $(this).prop("disabled", false);
-        return false
-      }
+        if (bgDate == ''){
+          bgDateInput.focus()
+          $('.form-control').each(function() {
+                bgDateInput[0].setCustomValidity("Please enter a date");
+                bgDateInput[0].reportValidity();
+          });
+          $(this).prop("disabled", false);
+          return false
+        }
 
-      let data = {
-          bgStatus: bgStatus,      // Expected to be one of the three background check statuses
-          user: $(this).data("username"),   // Expected to be the username of a volunteer in the database
-          bgType: $(this).attr("id"),       // Expected to be the ID of a background check in the database
-          bgDate: bgDate  // Expected to be the date of the background check completion or '' if field is empty
-      }
-      $.ajax({
-        url: "/addBackgroundCheck",
-        type: "POST",
-        data: data,
-        success: function(s){
-          var date = new Date(data.bgDate + " 12:00").toLocaleDateString()
-          reloadWithAccordion("background")
-        },
-        error: function(error, status){
-            console.log(error, status)
+        let data = {
+            bgStatus: bgStatus,      // Expected to be one of the three background check statuses
+            user: $(this).data("username"),   // Expected to be the username of a volunteer in the database
+            bgType: $(this).attr("id"),       // Expected to be the ID of a background check in the database
+            bgDate: bgDate  // Expected to be the date of the background check completion or '' if field is empty
         }
-      })
-  });
+        $.ajax({
+          url: "/addBackgroundCheck",
+          type: "POST",
+          data: data,
+          success: function(s){
+            var date = new Date(data.bgDate + " 12:00").toLocaleDateString()
+            msgFlash(`Successfully added background check`, "success", 1300)
+            setTimeout(function() {
+            reloadWithAccordion("background")
+            }, 1500)
```

Comment:
Same detail here, instead of delaying reload we could just show the flash after reload